### PR TITLE
update links and 2.1 typo

### PR DESCRIPTION
**Pull request by a-mccarthy**
Signed-off-by: Abigail McCarthy <mabigail@vmware.com>

### CODE REVIEW

File: docs/administration/vulnerability-scanning/import-vulnerability-data.md [hunk 1]
```diff
@@ -62,4 +62,4 @@ docker exec harbor-db /bin/sh -c  "pg_dump -U postgres -c" > all.sql
 
 ## Rescan the Images
 
-After importing the data, trigger the scanning process in the Harbor interface. For information about running a scan, see [Scan All Images](scan-all-images.md).
+After importing the data, trigger the scanning process in the Harbor interface. For information about running a scan, see [Scan All Images](scan-all-artifacts).
```
Comment (jonasrosland) [inline docs/administration/vulnerability-scanning/import-vulnerability-data.md:65]: Can we change the link to be Artifacts as well?

File: docs/install-config/configure-internal-tls.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This functionality is introduced via the `internal_tls` in `harbor.yml` file. To
 
 All certs can be automatically generated by `prepare` tool.
 ```bash
-docker run -v /:/hostfs goharbor/prepare:v2.0 gencert -p /path/to/internal/tls/cert
+docker run -v /:/hostfs goharbor/prepare:v2.1 gencert -p /path/to/internal/tls/cert
```
Comment (jonasrosland) [inline docs/install-config/configure-internal-tls.md:12]: Should this be `goharbor/prepare:v2.1.0` ?

File: docs/working-with-projects/working-with-images/preheat-images.md [hunk 3]
```diff
@@ -2,12 +2,11 @@
 title: Preheat Images
 weight: 30
 ---
-{{< note >}}
+
 Before you can preheat images, a system administrator must first configure a P2P provider instance. See more about [configuring P2P preheating in Harbor](../../administration/p2p-preheat/manage-preheat-providers.md).
```
Comment (jonasrosland) [inline docs/working-with-projects/working-with-images/preheat-images.md:6]: This link is broken, should go to administration/p2p-preheat/manage-preheat-providers/

**Review by jonasrosland — CHANGES_REQUESTED** [review]
This looks great, just a few small changes needed

File: docs/working-with-projects/working-with-images/preheat-images.md (hunk 3 above)
Comment (a-mccarthy) [reply docs/working-with-projects/working-with-images/preheat-images.md]: updated

File: docs/install-config/configure-internal-tls.md (hunk 2 above)
Comment (a-mccarthy) [reply docs/install-config/configure-internal-tls.md]: yup!

File: docs/administration/vulnerability-scanning/import-vulnerability-data.md (hunk 1 above)
Comment (a-mccarthy) [reply docs/administration/vulnerability-scanning/import-vulnerability-data.md]: we sure can! :)

**Review by jonasrosland — APPROVED** [review]
LGTM!

**Pull request merged**